platform-sdk/swirlds-platform-core/src/test/resources/com/swirlds/platform/config/legacy/empty.txt

```diff
@@ -0,0 +1 @@
+nextnodeid, 0
```

Comment:
Not a hard requirement, just thinking aloud. Would it be nicer to use `nextNodeId`? The other keywords are all lower case, but they are also all a single word.